src/snowflake/snowpark/dataframe.py

```diff
@@ -5613,6 +5615,23 @@ def dtypes(self) -> List[Tuple[str, str]]:
         ]
         return dtypes
 
+    def toJSON(self) -> "DataFrame":
+        """
```

Comment:
a nitpick but I also feel like we should add to_json and add alias for `toJSON`. This has been pattern in snowpark repo. I don't like this practice right now but for migration purposes it is useful